### PR TITLE
Updated Auth page: Increased the size between the fields.

### PR DESCRIPTION
**Pull request by Visham32**

### CODE REVIEW

File: app/styles/main.css [hunk 1]
```diff
@@ -197,3 +197,11 @@ body {
   text-overflow: ellipsis;
   margin-bottom: 20px;
 }
+
+label,input {
```
Comment (jaylenw) [inline app/styles/main.css:201]: Lets move this after 
 body {
}

and add spacing between label and input like:

label, input {

}

File: app/styles/main.css [hunk 2]
```diff
@@ -197,3 +197,11 @@ body {
...
+display: block;
+}
+
+label{
+margin-bottom: 10px;
```
Comment (jaylenw) [inline app/styles/main.css:206]: move this as well after body like what my comment above explained.

Also, be mindful of the spacing. Basically want the indentation to line up like everything else in the css file

label {
    margin-bottom: 10px;
}

**Review by jaylenw — COMMENTED** [review]
For some reason this branch is causing me issues when merging in. Go to the original pull request for this feature and add the changes i commented in here to there #39

**Comment by jaylenw** [comment]
For some reason, when you resolved your merge conflict issue, I am getting an extra copy of the AngularjsTodo App folder. I am going to close this pull request and you should make the changes you did here with my recommendations on the this pull request here #39  Thank you!

**Comment by Visham32** [comment]
The pull request #39 is does not have the file main.css involved. Is it ok if i create another pull request? @jaylenw

**Comment by jaylenw** [comment]
@Visham32 When you check out the branch you made for pull request #39, and make changes to the css file and commit it, then push it up to github, the file will be added to pull request #39 . So there is no need to create another pull request.

**Comment by Visham32** [comment]
I just did the required changes in the pull request #39

**Comment by jaylenw** [comment]
I will review #39 shortly.

**Pull request closed without merging**